Source/Core/DolphinQt/Config/Mapping/WiimoteEmuExtension.cpp

```diff
@@ -201,6 +203,19 @@ void WiimoteEmuExtension::CreateUDrawTabletLayout()
   m_udraw_tablet_box->setLayout(hbox);
 }
 
+void WiimoteEmuExtension::CreateTaTaConLayout()
+{
+  auto* hbox = new QHBoxLayout();
+  m_tatacon_box = new QGroupBox(tr("TaTaCon"), this);
+
+  hbox->addWidget(CreateGroupBox(
+      tr("Center"), Wiimote::GetTaTaConGroup(GetPort(), WiimoteEmu::TaTaConGroup::Center)));
+  hbox->addWidget(CreateGroupBox(
+      tr("Rim"), Wiimote::GetTaTaConGroup(GetPort(), WiimoteEmu::TaTaConGroup::Rim)));
```

Comment:
Do we need `i18n` comments at every occurrence of the word, or just one of them?

Reply:
Just one should be fine.